themes/bootstrap5/templates/cart/cart.phtml

```diff
@@ -3,7 +3,11 @@
   $this->headTitle($this->translate('Book Bag'));
 
   // Set up breadcrumbs:
-  $this->layout()->breadcrumbs = $this->searchMemory()->getLastSearchLink($this->transEsc('Search'), '<li class="breadcrumb-item">', '</li> ')
+  $this->breadcrumbs()->reset();
+   if ($url = $this->searchMemory()->getLastSearchUrl()) {
+     $this->breadcrumbs()->add($this->translate('Search'), $url);
+   }
+   $this->breadcrumbs()->add($this->translate('Cart'), $this->url('cart-home'));
```

Comment:
Indentation got out of alignment on a bunch of these. I'll suggest corrections here and below. This is very possibly VSCode's fault; sometimes pasting code into it causes it to make strange decisions about indentation.

```suggestion
  $this->breadcrumbs()->reset();
  if ($url = $this->searchMemory()->getLastSearchUrl()) {
    $this->breadcrumbs()->add($this->translate('Search'), $url);
  }
  $this->breadcrumbs()->add($this->translate('Cart'), $this->url('cart-home'));
```